src/Serilog/LoggerConfiguration.cs

```diff
@@ -39,6 +39,7 @@ public class LoggerConfiguration
         LogEventLevel _minimumLevel = LogEventLevel.Information;
         LoggingLevelSwitch _levelSwitch;
         int _maximumDestructuringDepth = 10;
+        int _maximumStringLength = 1000;
```

Comment:
So that this doesn't need to be flagged as a breaking change, perhaps `int.MaxValue` is a better option here?